### PR TITLE
Add a link which showcases a list of realworld UIs using Tailwind

### PR DESCRIPTION
**Pull request by asvny**
In this repo, I rebuild UIs of Shopify, Spotify, Netlify, and Atlassian using Tailwind.css.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -193,7 +193,8 @@
 
 - 🧪 [Tailwind Dark Mode Theme Switcher](https://github.com/huphtur/tailwind-theme-switcher) - Switching themes with CSS Custom Properties and Tailwind CSS.
 - 🧪 [Plugin Examples (Official)](https://github.com/tailwindcss/plugin-examples) - Official plugin examples.
-- 🧪 [Acquia](https://github.com/opdavies/rebuilding-acquia) - Acquia’s hosting dashboard rebuilt with Vue.js and Tailwind CSS.
```
Comment (joshmanders) [inline readme.md]: Please fix this, you broke this link.

Comment (asvny) [reply readme.md]: My bad . Sorry, I have fixed it now.

File: readme.md [hunk 2]
```diff
@@ -218,6 +218,7 @@
 - 🎬 [Let’s Build: Dribbble Shot](https://web-crunch.com/lets-build-tailwind-css-dribbble-shot) - Dribbble shot with Tailwind CSS.
 - 🎬 [Let’s Build: Tweet component](https://web-crunch.com/lets-build-tailwind-css-tweet) - Building a Tweet component with Tailwind CSS.
 - 🎓 [Modal Dialog](https://codeburst.io/creating-a-modal-dialog-with-tailwind-css-42722c9aea4f) - Creating a modal dialog with Tailwind CSS.
+- 🎓 [Building realworld UI using Tailwind.css](https://github.com/asvny/building-realworld-user-interfaces-using-tailwind) - Examples of building UIs of Shopify, Spotify, Netlify and Atlassian using Tailwind.css
```
Comment (hawezo) [inline readme.md]: Can you change `realworld` to `real-world` and `UI` to `UIs` in the title, and change `Tailwind.css` to `Tailwind CSS` in the description? As well as adding a dot at the end. Thanks!

Comment (asvny) [reply readme.md]: I have made the requested changes.

Comment (hawezo) [reply readme.md]: Thanks, and my bad, the first `Tailwind.css` slipped through my review. Can you change it to `Tailwind CSS`? After that, I'll approve it. 

(Also, it's not nitpicking but for consistency's sake, and I'd like this repository to live up to Tailwind's quality)

Comment (asvny) [reply readme.md]: @hawezo I have fixed it. Yeah, I was not sure what to type, so I went with _nitpick_  😇. Sorry, if it has hurt you.

Comment (joshmanders) [reply readme.md]: No worries, we're just trying to keep things clean and consistent. Thanks for the change!

**Pull request merged**